[package] mingw-w64/8.0.2: Using mingw from system instead of from the package

### Description

[conan-install-mingw-w64.log](https://github.com/user-attachments/files/17525559/conan-install-mingw-w64.log)

recipe fails during compilation of mingw-w64-crt ^

also, recipe will fail during gcc compilation if LIBRARY_PATH or LD_LIBRARY_PATH contains an empty entry, like this:
```
$ echo $LIBRARY_PATH
:/usr/lib/nvidia-367
```



### Package and Environment Details

* Package Name/Version: **mingw-w64/8.0.2**
* Operating System+version: **Linux Ubuntu 20.04**
* Compiler+version: **GCC 9**
* Conan version: **conan 2.8.1**
* Python version: **Python 3.8.20**


### Conan profile

local default profile:
```
[settings]
arch=x86_64
build_type=Release
compiler=gcc
compiler.cppstd=gnu14
compiler.exception=dwarf2
compiler.libcxx=libstdc++11
compiler.version=9
compiler.threads=posix
os=Linux
os.distribution=Ubuntu
os.distribution.version=20.04
```


### Steps to reproduce

```
$ cat /tmp/conanfile.txt 
[requires]
mingw-w64/8.0.2

$ conan install --build=missing /tmp/conanfile.txt 
```


### Logs

<details><summary>Click to expand log</summary>

```
  CC       stdio/lib32_libucrtbase_a-ucrt_vfprintf.o
/home/pkoshevoy/.conan2/p/b/mingw80cee71e4f445/b/src/mingw-w64/mingw-w64-crt/stdio/ucrt_sprintf.c: In function ‘sprintf’:
/home/pkoshevoy/.conan2/p/b/mingw80cee71e4f445/b/src/mingw-w64/mingw-w64-crt/stdio/ucrt_sprintf.c:16:33: error: ‘_CRT_INTERNAL_PRINTF_STANDARD_SNPRINTF_BEHAVIOR’ undeclared (first use in this function); did you mean ‘UCRTBASE_PRINTF_STANDARD_SNPRINTF_BEHAVIOUR’?
   16 |   ret = __stdio_common_vsprintf(_CRT_INTERNAL_PRINTF_STANDARD_SNPRINTF_BEHAVIOR, _Dest, (size_t)-1, _Format, NULL, ap);
      |                                 ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      |                                 UCRTBASE_PRINTF_STANDARD_SNPRINTF_BEHAVIOUR
/home/pkoshevoy/.conan2/p/b/mingw80cee71e4f445/b/src/mingw-w64/mingw-w64-crt/stdio/ucrt_sprintf.c:16:33: note: each undeclared identifier is reported only once for each function it appears in
  CC       stdio/lib32_libucrtbase_a-ucrt_vprintf.o
/home/pkoshevoy/.conan2/p/b/mingw80cee71e4f445/b/src/mingw-w64/mingw-w64-crt/stdio/ucrt__vscprintf.c:11:5: error: redefinition of ‘_vscprintf’
   11 | int _vscprintf(const char * __restrict__ _Format, va_list _ArgList)
      |     ^~~~~~~~~~
In file included from /home/pkoshevoy/.conan2/p/b/mingw80cee71e4f445/b/src/mingw-w64/mingw-w64-crt/stdio/ucrt__vscprintf.c:9:
/usr/share/mingw-w64/include/stdio.h:946:15: note: previous definition of ‘_vscprintf’ was here
  946 |   int __cdecl _vscprintf(const char * __restrict__ _Format,va_list _ArgList)
      |               ^~~~~~~~~~
```
</details>

--- a/recipes/mingw-w64/linux/conanfile.py
+++ b/recipes/mingw-w64/linux/conanfile.py
@@ -118,7 +118,7 @@ class MingwConan(ConanFile):
 
         # Add binutils to path. Required for gcc build.
         env = Environment()
-        env.append_path("PATH", os.path.join(self.package_folder, "bin"))
+        env.prepend_path("PATH", os.path.join(self.package_folder, "bin"))
         env.vars(self).save_script("conanbuild_package_bin_path")
 
         venv = VirtualBuildEnv(self)

